docker-compose-postgres.yml

```diff
@@ -0,0 +1,66 @@
+services:
+  postgres:
+    image: postgres:14
+    container_name: plone-postgres
+    ports:
+      - "127.0.0.1:5432:5432"
+    environment:
+      POSTGRES_USER: plone
+      POSTGRES_DB: local
+      POSTGRES_PASSWORD: qweqwe1!
+    volumes:
+      - type: bind
+        source: ./plone-postgres-data/
+        target: /var/lib/postgresql/data
+  plone:
+    image: plone-postgres-local:latest
```

Comment:
if we make this the default docker-compose approach, this image name can probably be simplified back to plone or something (just the postgres reference becomes redundant)